docs/reference/installation.rst

```diff
@@ -215,62 +214,120 @@ Routing Configuration
-
-    return [
-        // ...
-        App\Application\Sonata\PageBundle\ApplicationSonataPageBundle::class => ['all' => true],
-    ];
-
-Configure SonataPageBundle to use the newly generated classes:
+    # config/packages/doctrine.yaml
 
-.. code-block:: yaml
+    doctrine:
+        orm:
+            entity_managers:
+                default:
+                    mappings:
+                        SonataPageBundle: ~
 
-    # config/packages/sonata_page.yaml
+And then create the corresponding entities, ``src/Entity/SonataPageBlock``::
+
+    // src/Entity/SonataPageBlock.php
+
+    use Doctrine\ORM\Mapping as ORM;
+    use Sonata\PageBundle\Entity\BaseBlock;
+
+    /**
+     * @ORM\Entity
+     * @ORM\Table(name="page__block")
+     */
+    class SonataPageBlock extends BaseBlock
+    {
+        /**
+         * @ORM\Id
+         * @ORM\GeneratedValue
+         * @ORM\Column(type="integer")
+         */
+        protected $id;
+    }
+
+``src/Entity/SonataPagePage``::
+
+    // src/Entity/SonataPagePage.php
+
+    use Doctrine\ORM\Mapping as ORM;
+    use Sonata\PageBundle\Entity\BasePage;
+
+    /**
+     * @ORM\Entity
+     * @ORM\Table(name="page__page")
+     */
+    class SonataPagePage extends BasePage
+    {
+        /**
+         * @ORM\Id
+         * @ORM\GeneratedValue
+         * @ORM\Column(type="integer")
+         */
+        protected $id;
+    }
+
+``src/Entity/SonataPageSite``::
+
+    // src/Entity/SonataPageSite.php
+
+    use Doctrine\ORM\Mapping as ORM;
+    use Sonata\PageBundle\Entity\BaseSite;
+
+    /**
+     * @ORM\Entity
+     * @ORM\Table(name="page__site")
+     */
+    class SonataPageSite extends BaseSite
+    {
+        /**
+         * @ORM\Id
+         * @ORM\GeneratedValue
+         * @ORM\Column(type="integer")
+         */
+        protected $id;
+    }
+
+and ``src/Entity/SonataPageSnapshot``::
+
+    // src/Entity/SonataPageSnapshot.php
+
+    use Doctrine\ORM\Mapping as ORM;
+    use Sonata\PageBundle\Entity\BaseSnapshot;
+
+    /**
+     * @ORM\Entity
+     * @ORM\Table(name="page__snapshot")
+     */
+    class SonataPageSnapshot extends BaseSnapshot
+    {
+        /**
+         * @ORM\Id
+         * @ORM\GeneratedValue
+         * @ORM\Column(type="integer")
+         */
+        protected $id;
+    }
+
+The only thing left is to update your schema::
 
-    sonata_page:
-        class:
-            page: App\Application\Sonata\PageBundle\Entity\Page # This is an optional value
-            snapshot: App\Application\Sonata\PageBundle\Entity\Snapshot
-            block: App\Application\Sonata\PageBundle\Entity\Block
-            site: App\Application\Sonata\PageBundle\Entity\Site
+    bin/console doctrine:schema:update --force
 
-The only thing left is to update your schema:
+Next Steps
+----------
 
-.. code-block:: bash
+At this point, your Symfony installation should be fully functional, with no errors
+showing up from SonataPageBundle. If, at this point or during the installation,
```

Comment:
If you come across any error, at this point of during the installation, don't panic: